salt/modules/oracle.py

```diff
@@ -37,6 +40,11 @@
 import os
 import logging
 from salt.utils.decorators import depends
+try:
+    from salt.utils.files import fopen
+except ImportError:
+    from salt.utils import fopen
```

Comment:
Why would this be necessary, considering this is going into the develop branch of Salt?

Reply:
hey mike, you're absolutely right. please let me set up an aix minion with salt from dev branch so that i can make a real test of the proposed changes. until now i've based this on stable minion 2016-xx. i will have the next update til friday 23rd. also.. apart from the import is there anything else that needs to be looked at?

Reply:
This was my only concern. Thanks, @jose1711

Reply:
fixed and tested on a real db system with develop branch